File: Lab2/javaServer/src/main/java/com/JabaJabila/javaServer/dto/CatCreationInfo.java
package com.JabaJabila.javaServer.dto;

import com.JabaJabila.javaServer.entities.CatColor;

import java.util.Date;

public class CatCreationInfo {
    private Long catId;
    private String name;
    private CatColor color;
    private String breed;
    private Date birthdate;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public CatColor getColor() {
        return color;
    }

    public void setColor(CatColor color) {
        this.color = color;
    }

    public String getBreed() {
        return breed;
    }

    public void setBreed(String breed) {
        this.breed = breed;
    }

    public Date getBirthdate() {
        return birthdate;
    }

    public void setBirthdate(Date birthdate) {
        this.birthdate = birthdate;
    }
}

File: Lab2/Codegen/MyHttpClient/GeneratedModels/CatCreationInfo.cs
namespace MyHttpClient.GeneratedModels;
public class CatCreationInfo
{
    public CatCreationInfo(string name, string breed, string birthdate)
    {
        Name = name;
        Breed = breed;
        Birthdate = birthdate;
    }

    public string Name { get; set; }

    public string Breed { get; set; }

    public string Birthdate { get; set; }
}